yaml/xyz/openbmc_project/Logging/AuditLog.interface.yaml

```diff
@@ -0,0 +1,43 @@
+description: >
+    This interface provides methods to extract the Linux audit logged events.
+methods:
+    - name: ParseAuditLog
+      description: >
+          Parses all previously recorded log events into JSON format. Returns a
+          read-only file descriptor to the parsed file. Entries are sorted
+          oldest to newest.
+      returns:
+          - name: logFd
+            type: unixfd
+            description: >
+                The read-only file descriptor of the JSON formatted log events.
+      errors:
+          - xyz.openbmc_project.Common.Error.InternalFailure
+          - xyz.openbmc_project.Common.File.Error.Write
+          - xyz.openbmc_project.Common.File.Error.Open
+
+    - name: GetAuditLog
+      description: >
+          Parses a subset of previously recorded log events into JSON format.
+          Returns a read-only file descriptor to the parsed file. Entries are
+          sorted newest to oldest.
+      parameters:
+          - name: maxCount
+            type: uint32
+            description: Maximum number of entries to return.
```

Comment:
Is there a max here?

Reply:
I took a lot of my questions by looking at top Redfish query parameter 

```

$top=<integer> | Applies to Resource Collections.  Defines the number of members to show in the response.Minimum value is 1.  By default, returns all members.Example:http://resourcecollection?$top=30

```

Reply:
Good point, I will update to expand on the description.

(I don't have a default value defined. bmcweb does have a default top value which is what I'm passing in if the Redfish path didn't specify a top value. But I could define a default value here if that seems better.

I'm also not adding default into the description since with sdbusplus there would be a "default:" line in the YAML description of this parameter if it existed.)

"Defines the maximum number of entries to return. Minimum value is 1."